packages/11ty/_tests/helpers/index.js

```diff
@@ -0,0 +1,40 @@
+import Eleventy from '@11ty/eleventy'
+
+/**
+ * @function stubGlobalData
+ *
+ * @param {Object} eleventyConfig
+ *
+ * Inserts keys / values into globalData suitable to make Eleventy() init
+ *
+ **/
+const stubGlobalData = (eleventyConfig) => {
+  eleventyConfig.addGlobalData('publication', {})
+
+  const config = {
+    accordion: {
+      copyButton: {}
+    },
+    epub: {},
+    figures: {}
+  }
+  eleventyConfig.addGlobalData('config', config)
+
+  const figures = { figure_list: [] }
+  eleventyConfig.addGlobalData('figures', figures)
+}
+
+/**
+ * @function initEleventyEnvironment
+ *
+ * Initializes an Eleventy object suitable for rendering out shortcodes
+ *
+ **/
+const initEleventyEnvironment = async () => {
+  const elev = new Eleventy('../', '_site', { config: stubGlobalData })
+  await elev.init()
+
+  return elev
+}
```

Comment:
⚙️ prefer more rather than less explicit variable names
```suggestion
const initEleventyEnvironment = async () => {
  const eleventy = new Eleventy('../', '_site', { config: stubGlobalData })
  await eleventy.init()
  return eleventy
}
```